group.py

```diff
@@ -1,5 +1,39 @@
 """An example of how to represent a group of acquaintances in Python."""
 
-# Your code to go here...
+entries = [
+    ["Jill", 26, "biologist", {"friend":"Zalika","partner":"John"}],
+    ["Zalika", 28, "artist", {"friend":"Jill"}]
+           ]
 
-my_group =
+my_group = [
+    {'name': 'Jill', 'age': 26, 'job': 'biologist', 'relation': {'friend': 'Zalika', "partner":"John"}}, 
+    {'name': 'Zalika', 'age': 28, 'job': 'artist', 'relation': {'friend': 'Jill'}}
+    ]
+
+"""
+def write_entries(entries):
```

Comment:
A rather complicated way of doing this task/ not easily readable. The function is redundant in this case because you still have to write out the entries in the dictionary.